code/modules/admin/verbs/trading_verbs.dm

```diff
@@ -1,5 +1,5 @@
 /client/proc/list_traders()
-	set category = "Debug"
+	set category = null
```

Comment:
Just delete these if they're unsupported, then.

Reply:
just remove all mentions of `/client/proc/list_traders` in the code and it won't break anything